package.json

```diff
@@ -39,6 +39,12 @@
     "build:assets:other": "cpy --flat 'src/assets' '!**.svg' '!**.tsx' assets",
     "build:assets:svg": "svgo --folder src/assets -o assets",
     "build:react": "tsup src/react/index.ts --dts --format=esm,cjs --sourcemap --out-dir=react",
+    "build:src": "yarn run build:src:logo && yarn run build:src:favicon:png && yarn run build:src:favicon:ico && yarn run build:src:android-icon && yarn run build:src:apple-touch-icon",
+    "build:src:android-icon": "convert src/assets/favicon.png \\( +clone -resize 192x192 -write src/assets/icon-192.png +delete \\) -resize 512x512 src/assets/icon-512.png",
+    "build:src:apple-touch-icon": "rsvg-convert src/assets/favicon.svg -f png -a -w 140 -h 140 | convert - -background none -gravity center -extent 180x180 src/assets/apple-touch-icon.png",
+    "build:src:favicon:ico": "convert src/assets/favicon.png -resize 48x48 src/assets/favicon.ico",
```

Comment:
Should the favicon be 32⨯32?

Reply:
So, according to the "favicons in 2023" guide you posted, yes. The favicon that was in the repo prior to my changes was 48x48 so I just kept it the same.

It's an easy change to make it 32x32 in accordance with that guide. I looked around more and 32x32 seems consistently the browser tab resolution.

Reply:
Ahh, I didn't realize that. I'll create a separate issue for that.